### PR TITLE
[Bugs] Changes on explore tools, tooltip banner

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description
 
- New layout for explore tools on desktop
- Fix Url for tooltip banner 

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: src/components/TooltipBanner/index.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ function TooltipBanner({ tagline, ctaText, ...props }) {
             <Button
               variant="text"
               component={Link}
-              href="/explore?showTutorial=1"
+              href="/explore/ke?showTutorial=1"
```
[the remark this answers is not in the document]
Comment (KhadijaMahanga) [reply src/components/TooltipBanner/index.js:41]: @kilemensi woii!! the whole API is down

[the remark this answers is not in the document]
Comment (KhadijaMahanga) [reply src/components/TooltipBanner/index.js:41]: @kilemensi the whole explore page is down

File: src/components/ProjectPartners/index.js [hunk 2]
```diff
@@ -60,7 +60,11 @@ function ProjectPartners({ title, partners, ...props }) {
           </Typography>
         </Grid>
         {partners?.map(({ link, logo, logoProps, name }, index) => (
-          <Grid key={link} item md={index > 1 ? 4 : 6}>
+          <Grid
+            key={link}
+            item
+            md={index > 1 && partners.length - index > 2 ? 4 : 6}
```
[the remark this answers is not in the document]
Comment (KhadijaMahanga) [reply src/components/ProjectPartners/index.js]: @kilemensi This wasn't supposed to be committed.

**Review by kilemensi — APPROVED** [review]
🚀 

---

[Youth's Voter Registration Statistics](https://pesayetu-git-fixes-homepage-codeforafrica.vercel.app/stories/insights/youths-voter-registration-statistics) chart returns 404

**Pull request merged**